alternator/executor.cc

```diff
@@ -3160,7 +3160,10 @@ future<> executor::do_batch_write(
             key_builders(1, schema_decorated_key_hash{}, schema_decorated_key_equal{});
         for (auto& b : mutation_builders) {
             auto dk = dht::decorate_key(*b.first, b.second.pk());
-            auto [it, added] = key_builders.try_emplace(schema_decorated_key{b.first, dk});
+            auto [it, added] = key_builders.try_emplace(schema_decorated_key {
+                .schema = std::move(b.first),
+                .dk = std::move(dk)
+            });
```

Comment:
Looks correct, because indeed we don't touch mutation_builders later (and we will now need to remember not to do this any more).
I wonder if this should be made more "official" by replacing the loop by `for (auto&& b : mutation_builders) {` or even `for (auto&& b : std::move(mutation_builders)) {`.